### DISCUSSION TITLE
Rollup Error when building storybook but nothing when running

### DISCUSSION THREAD

**Discussion by bastiengrignon** [storybookjs/storybook#25885] [Help]
### Summary

After upgrading storybook, I cannot build it anymore I got a RollupError (cf. screenshot), but I got nothing when running Storybook I got no error.

The error doesn't seems to be linked to my code  so I don't really how can I resolve it :disappointed: 

### Additional information

![Screen at 02-02-2024_12:16:51](https://github.com/storybookjs/storybook/assets/41306995/08dcb061-b4dc-4e14-a49b-e370bea47c14)

Here is my `main.js` config
```js
export default {
  stories: ['../src/**/*.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
  addons: [
    '@storybook/addon-links',
    '@storybook/addon-essentials',
    '@storybook/addon-interactions',
    '@storybook/addon-a11y',
    'storybook-dark-mode',
    '@storybook/addon-styling',
    {
      name: '@storybook/addon-docs',
      options: {
        mdxPluginOptions: {
          mdxCompileOptions: {
            remarkPlugins: [remarkGfm],
          },
        },
      }
    }
  ],
  framework: {
    name: '@storybook/react-vite',
    options: {}
  },
  features: {
    storyStoreV7: true
  },
  viteFinal: async (config, {
    configType
  }) => ({
    ...config,
    optimizeDeps: {
      ...config.optimizeDeps,
      include: ['storybook-dark-mode']
    },
    plugins: configType === 'PRODUCTION' ? [...config.plugins, turbosnap({
      rootDir: config.root ?? process.cwd()
    })] : config.plugins,
    resolve: {
      alias: aliases
    }
  }),
  docs: {
    autodocs: true
  }
};
```

**Comment by benjazehr** [comment]
I suspect it has something to do with `storybook-dark-mode`. I ran into the same issue and when I disabled everything related to that addon, the error disappeared.

**Comment by valentinpalkovic** [comment]
Hi @bastiengrignon 

Thanks for reaching out!
It seems that `storybook-dark-mode` might be the culprit. Indeed, Storybook 8 still needs to be supported by the addon. You would have to track this [PR](https://github.com/hipstersmoothie/storybook-dark-mode/pull/251) to keep up to date and know when it is finally ready to support Storybook 8.

**Comment by soullivaneuh** [comment]
For my case, I fall into trouble because of `@whitespace/storybook-addon-html` addon. See the related report on the vendor issue: https://github.com/chromaui/storybook-addon-pseudo-states/issues/108#issuecomment-1999477683
